### DISCUSSION TITLE
Will there be a version for Mac and Linux?

### DISCUSSION THREAD

**Discussion by cyb3rdog** [cyb3rdog/Cyb3rVector#7] [Q&A] [answered]
## Will there be a version for Mac and Linux? And when?

**Comment by cyb3rdog** [comment, marked as answer]
### Yes, it is in works! But it wont happen overnight.

The core, and majority of the Cyb3rVector has been already ported to platform independent code-base and successfully tested. The very last layer, which is still incomplete, is the User Inteface itself (the application's screens). This one requires a little more effort, as it about porting the WPF to MVX.

However, with a little help from some additional software, such as Virtual Machines, or Parallels, etc., you should be able to run this program already.
[Stage II. of the project's roadmap](https://cyb3rdog.github.io/Cyb3rVector/) also aims for the Online Codelab feature, hosted on my own servers. This however, is still quite far and working on project like this is time consuming process.

Please note, that I still have my regular job, and I am doing all this in my free personal time, for free of charge, purely from my passion for Vector and for this dream.

**Reply by moribundant** [reply]
I have it running on a MacBook Pro M1 under Parallels Windoze 10--works great. Would be better if native, but for now just fine.
